containers/buildUbuntu2.docker/scripts/dummy

```diff
@@ -0,0 +1 @@
+A file to keep the directory.
```

Comment:
If this is for Git, then the usual convention is to call the file `.gitkeep`

Reply:
+1